serverless/architecture/serverless-event-architecture.adoc

```diff
@@ -0,0 +1,39 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-architecture"]
+= Knative Eventing architecture
+:context: serverless-event-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Eventing on {product-title} enables developers to use an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] with serverless applications.
+An event-driven architecture is based on the concept of decoupled relationships between event producers that create events, and event  _sinks_, or consumers, that receive them.
+
+Knative Eventing uses standard HTTP POST requests to send and receive events between event producers and consumers.
+These events conform to link:https://cloudevents.io[CloudEvents specifications], which enables creating, parsing, sending, and receiving events in many programming languages.
+Supported languages include Go, JavaScript, Java, and Python.
```

Comment:
There are more languages supported by CloudEvents SDKs (C#, Ruby, Rust), and it's also possible to use cloud events in any language by following specification. SDK gives developers an easier way to start using cloud-events/eventing.

I wrote about only Go, JavaScript, Java, and Python as I presume those are the ones that Red Hat is focussing on.

Maybe:
```suggestion
These events conform to link:https://cloudevents.io[CloudEvents specifications], which enables creating, parsing, sending, and receiving events in many programming languages.
Supported languages include Go, JavaScript, Java, and Python.
These events conform to link:https://cloudevents.io[CloudEvents specifications], which enables creating, parsing, sending, and receiving events in any programming language.
_CloudEvents_ provides also a convinient SDKs for languages many like Go, JavaScript, Java, Python etc.
```

Reply:
Removing the line about the languages if it's basically all languages. There's no need for us to list them all. What we can do instead maybe is add examples for different languages someday in the future.

Reply:
Specification let you implement CloudEvents in **"any"** language. Now, there's a **"many"** word being used. It's different meaning.